internal/kaniko/execute.go

```diff
@@ -47,9 +63,139 @@ func (k *Config) Run(ctx context.Context) (err error) {
 			return err
 		}
 	}
+
+	if k.SendArtifactInfo {
+		err = k.createArtifactInfo(k.client, k.processDestinations())
+		if err != nil {
+			log.Printf("WARN: failed to create artifact info: %v", err)
```

Comment:
using `%w` in the formatting for error will capture the full error message.

Reply:
@pinaki124 it seems that we can not use %w here because of:
`This verb can be used only in 'fmt.Errorf' calls (%w)`

Reply:
you are correct, i missed the fact that this is using the `log.Printf` method - you can keep it as `%v` then.